glib/src/functions.rs

```diff
@@ -27,10 +27,17 @@ use std::ptr;
 #[cfg_attr(feature = "dox", doc(cfg(all(feature = "v2_58", not(windows)))))]
 #[allow(clippy::too_many_arguments)]
 #[doc(alias = "g_spawn_async_with_fds")]
-pub fn spawn_async_with_fds<P: AsRef<std::path::Path>, T: AsRawFd, U: AsRawFd, V: AsRawFd>(
+pub fn spawn_async_with_fds<
+    P: AsRef<std::path::Path>,
+    T: AsRawFd,
+    U: AsRawFd,
+    V: AsRawFd,
+    A: AsRef<str>,
+    E: AsRef<str>,
+>(
     working_directory: P,
-    argv: &[&str],
-    envp: &[&str],
+    argv: &[A],
+    envp: &[E],
```

Comment:
Here.

Reply:
Those are all manually added, that's why I said that I need to change the manuals... ;) I will update the PR later